src/modules/ToDoList.js

```diff
@@ -0,0 +1,55 @@
+// import Task from './modules/Task.js';
+export default class ToDoList {
+  constructor(taskList = []) {
+    this.taskList = taskList;
+  }
+
+  get getTaskList() {
+    return this.taskList;
+  }
+
+  set setTaskList(newTaskList = []) {
+    if (!newTaskList) return;
+    this.taskList = newTaskList;
+  }
+
+  getLength() {
+    return this.taskList.length;
+  }
+
+  addTask(task) {
+    this.taskList.push(task);
+  }
+
+  removeTask(index) {
+    this.taskList.splice(index, 1);
+  }
+
```

Comment:
- [ ] Good job :+1: :100: implementing some of the functionalities required for the project. So, I noticed that you still miss one of them, the one of **Editing** a task description. For this, you should make sure once you have edited a task description, all changes you made are saved/stored in the **Local Storage**. Right now the ones you have are only simple `inputs`